spec/javascripts/reducers/project_spec.js

```diff
@@ -0,0 +1,36 @@
+import actionsTypes from "../../../app/assets/javascripts/actions/actionTypes";
+import reducer from "../../../app/assets/javascripts/reducers/project";
+
+describe("Project Reducer",() => {
+  describe("RECEIVE_PROJECT",() => {
+    it("returns action.data",() => {
```

Comment:
I believe `returns data from action` would be a better description for a test. 🤔 
What do you think?

Reply:
Yeah.... now that you mention, it's much better kk